packages/reakit-utils/src/getWindow.ts

```diff
@@ -3,6 +3,6 @@ import { getDocument } from "./getDocument";
 /**
  * Returns `element.ownerDocument.defaultView || window`.
  */
-export function getDefaultView(element?: Element): Window {
+export function getWindow(element?: Element): Window {
   return getDocument(element).defaultView || window;
```

Comment:
Should this also use a cached `_window` variable to avoid expensive window lookup and `typeof` usage?